apis/python/src/tiledbsoma/tdb_handles.py

```diff
@@ -81,7 +83,11 @@ def open(
     @classmethod
     @abc.abstractmethod
     def _opener(
-        cls, uri: str, mode: options.OpenMode, context: SOMATileDBContext
+        cls,
+        uri: str,
+        mode: options.OpenMode,
+        context: SOMATileDBContext,
+        auxiliary_reader: bool = False,
```

Comment:
It might make sense to call this `use_latest_read_timestamp` instead of `auxililary_reader`, so that it’s clearer what the effect, and add the comment about *why* we do it at line 71.